### DISCUSSION TITLE
Qt6 backend

### DISCUSSION THREAD

**Discussion by ghost** [slint-ui/slint#2488] [Q&A] [answered]
Is the QStyle backend limited to Qt5, or can I select Qt6? I'm targeting Wayland and default cargo features link against Qt5. https://github.com/slint-ui/slint/discussions/1389 sounds related and not at the same time.

**Comment by ogoffart** [comment, marked as answer]
You can set the `QMAKE` env variable to point to Qt6's qmake.

More info: https://docs.rs/qttypes/latest/qttypes/#finding-qt
